### ISSUE TITLE
TagPicker: does not read out the delete option of selected tag in edge

### ISSUE THREAD

**Issue by code-mage** [OfficeDev/office-ui-fabric-react#5098]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: High

Products/sites affected: VSTS

#### Describe the issue:

Tag Picker does not read out the delete option of selected tag in edge

#### Expected behavior:

Along with the name of the tag, delete option should also be mentioned in the name so that the user will be able to know that the tags can be deleted.

**Comment by betrue-final-final** [comment]
@lynamemi

**Comment by code-mage** [comment]
Guys, is there any progress on this?

**Comment by micahgodbolt** [comment]
duplicate of #4817
